tests/templates/System_Message.twig

```diff
@@ -0,0 +1 @@
+{$ message | trans() | raw $}
```

Comment:
I know the old application just displayed the message, but shouldn't we be a bit more user friendly and display it in the site layout?